### DISCUSSION TITLE
关于pytorch安装教程的问题

### DISCUSSION THREAD

**Discussion by qcname** [shenpengfii/shenpengfii.github.io#9] [Q&A] [answered]
报错
Downloading and Extracting Packages:

                                                                                                                  
CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/pytorch/w&gt; 
Elapsed: -                                                                                                        

An HTTP error occurred when trying to retrieve this URL.                                                          
HTTP errors are often intermittent, and a simple retry will get you on your way.                                  

CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/pytorch/w&gt; 
Elapsed: -

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.

命令：conda install pytorch=2.1.0=py3.11_cuda12.1_cudnn8_0  torchvision=0.16.0=py311_cu121  torchaudio=2.1.0=py311_cu121 pytorch-cuda=12.1 -c https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/pytorch/win-64 -c nvidia

**Comment by shenpengfii** [comment]
Have received your letter.

**Comment by shenpengfii** [comment, marked as answer]
我在本地用 Python 3.11 测试过了，但没办法复现这个问题。
但是有两个需要你注意的细节：
1. 在你提供的报错信息里，`<https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/pytorch/w>` 这个 URL 指向明显是不存在的，和你在“命令：”之后贴的 URL 不一致。最好看看是不是命令打错了。
2. Conda 环境的 HTTP `000` 错误代码一般是本机配置的问题导致的。你可以看看其他这方面的文章，比如这篇 [『技术随手学』解决CondaHTTPError: HTTP 000 CONNECTION 问题](https://zhuanlan.zhihu.com/p/260034241)。
